.github/workflows/jmh-benchmark.yml

```diff
@@ -0,0 +1,101 @@
+name: JMH Benchmark
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  setup:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/benchmark')
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Get PR branch
+      uses: xt0rted/pull-request-comment-branch@v1
+      id: comment-branch
+
+
+    - name: Set branch name
+      run: echo "BRANCH_NAME=${{ steps.comment-branch.outputs.head_ref }}" >> $GITHUB_ENV
+    
+    - id: 'auth'
+      name: Authenticating
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: '${{ secrets.GCP_SA_KEY }}'
+
+    - name: Set up Google Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+      
+      
+    - name: Setup Directory and Copy Bash Scripts
```

Comment:
I feel all these `gcloud` commands should be combined into a single shell script, and then we can have one step here that just runs the shell script.  This way, e.g., we can more easily add comments inside the script, handle failures, etc.

Reply:
I can merge all into one once the rest of the review process is done, so putting this suggestion off for the end.

Reply:
Why put it off?  It will make the set of commands easier to review, and it needs to be tested.  Please go ahead and make the change.